src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/RuntimeHelpersTests.cs

```diff
@@ -192,16 +191,102 @@ private static void FillStack(int depth)
             }
         }
 
+        public static IEnumerable<object[]> GetUninitializedObject_NegativeTestCases()
+        {
+            // TODO: Test actual function pointer types when typeof(delegate*<...>) support is available
```

Comment:
Is this TODO still relevant?

Reply:
Yeah. `typeof(delegate*<void>)` is basically treated the same as `typeof(IntPtr)`, so `GetUninitializedObject(typeof(delegate*<void>))` succeeds since it results in a zero-inited boxed `IntPtr`. Presumably once the runtime and reflection stack treat these as real pointer types then we'll want to block them, same as `void*` today.

Reply:
I would add this as an item to the issue about better support for pointers in reflection.

Reply:
> I would add this as an item to the issue about better support for pointers in reflection.

Already called out in the top post of #11354.